src/transformStateWithClones.js

```diff
@@ -6,8 +6,51 @@
  *
  * @return {Object[]}
  */
+
 function transformStateWithClones(state, actions) {
-  // write code here
+  // const newObject = {};
+
+  const clone = {}; // новий порожній об'єкт
```

Comment:
According to the naming conventions, the object copy should be named `stateCopy` instead of `clone` to make the code more readable and meaningful.